txt2man

```diff
@@ -45,40 +45,46 @@ DESCRIPTION
   Here is how text patterns are recognized and processed:
   Sections    These headers are defined by a line in upper case, starting
               column 1. If there is one or more leading spaces, a
-	      sub-section will be generated instead.
+              sub-section will be generated instead. Optionally, the
+              Section name can be preceded by a blank line. This is useful
+              for a better visualization of the source text to be used to
+              generate the manpage.
   Paragraphs  They must be separated by a blank line, and left aligned.
+              Alternatively two blank spaces can be used to produce the
+              same result. This option will provide a better visualization
+              of the source text to be used to generate the manpage.
   Tag list    The item definition is separated from the item description
               by at least 2 blank spaces, even before a new line, if
               definition is too long. Definition will be emphasized
               by default.
-  Bullet list  
+  Bullet list
```

Comment:
The 2 spaces before newline should be kept here: `"Bullet list  "` in order to be parsed as a tag list item.